src/dotnet/commands/dotnet-cache/LocalizableStrings.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Microsoft.DotNet.Tools.Cache
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET Cache";
+
+        public const string AppDescription = "Cahes Assemblies specified for the .NET Platform, By Default these will be optimized for the target runtime and framework";
+
+        public const string ProjectEntries = "ProjectEntries";
+
+        public const string ProjectEntryDescription = "The MSBuild project file used to specify the list of packages to be cached.";
+
+        public const string FrameworkOption = "FRAMEWORK";
+
+        public const string FrameworkOptionDescription = "Target framework for which to cache for";
+
+        public const string RuntimeOption = "RUNTIME_IDENTIFIER";
+
+        public const string RuntimeOptionDescription = "Target runtime to cache for.";
+
+        public const string OutputOption = "OUTPUT_DIR";
+
+        public const string OutputOptionDescription = "Path in which to cache the given assemblies";
```

Comment:
The other commands all describe this const as "Output directory in which to place...". Shouldn't this follow a similar pattern? Perhaps "Output directory in which to place the cached assemblies." or "Output directory in which to cache the given assemblies"